src/utilities/arrays.ts

```diff
@@ -3,13 +3,12 @@ import { floor, random } from '@/utilities/math'
 
 // we really do want any here
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function toRecord<T extends any[], K extends keyof T[number]>(source: T, key: K): Record<K, T> {
```

Comment:
This utility wasn't being used anywhere and I'm also pretty sure the types are incorrect for it anyway.